builder.go

```diff
@@ -42,6 +42,12 @@ func NewBuilder(c config) (*Bob, error) {
 	cmds := make([][]string, len(c.Script))
 	for i, s := range c.Script {
 		cmds[i] = strings.Split(s, " ")
+
+		// check for environment variables inside script
+		if !strings.Contains(s, "$$") {
+			continue
```

Comment:
Initially I had all the logic in here and didn't want all the nesting. I wouldn't mind moving replaceEnv to this line.